libs/Geometry3D/include/CLinkedVector3D.hpp

```diff
@@ -10,17 +10,35 @@ namespace Geometry3D {
      */
     class CLinkedVector3D : public AVector3D {
     private:
-        const CPoint3D &point_begin;
-        const CPoint3D &point_end;
+        const CPoint3D *point_begin = nullptr;
+        const CPoint3D *point_end = nullptr;
+
+        /**
+         * Set new begin point Rvalue. Cannot be used.
+         * @param new_point_begin Rvalue reference to begin point
+         */
+        void setPointBegin(CPoint3D&& new_point_begin) const noexcept;
+
+        /**
+         * Set new end point. Cannot be used.
+         * @param new_point_end Rvalue reference to end point
+         */
+        void setPointEnd(CPoint3D&& new_point_end) const noexcept;
+
+        /**
+         * Copy constructor RValue. Cannot be used.
+         * @param _point_begin Rvalue reference to begin point
+         * @param _point_end Rvalue reference to end point
+         */
+        CLinkedVector3D(CPoint3D &&_point_begin, CPoint3D &&_point_end);
     public:
         /**
          * Copy constructor Lvalue
-         * @param point_begin First point
-         * @param point_end Second point
+         * @param point_begin Lvalue reference to end point
```

Comment:
Replace 'Lvalue reference to end point' -> 'Lvalue reference to begin point'

Reply:
Complete